recipes/conflowgen/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "conflowgen" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7c35a8ff153277166fbd052156ee1b83279ec8b3a1b468fc4815a7cff3bc04c
+  # check hash at https://pypi.org/project/conflowgen/#files
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pandas >=1
+    - numpy
+    - peewee >=3
+    - enum_tools >=0.7
+    - matplotlib-base
+    - plotly
+    - seaborn
```

Comment:
the grayskull automated recipe generator suggested kaleido is needed. If so, you can add it as per my comment